### PR TITLE
Simple Clean-up

### PR DESCRIPTION
**Pull request by Nirmal4G**
- [x] Add Root EditorConfig

- [x] Remove BOM (Byte Order Mark) from all files

- [x] Format All files using VSCode's default formatters (except sources)

### CODE REVIEW

File: src/msc/msc.csproj [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <IsTool>true</IsTool>
     <OutputType>Exe</OutputType>
+    <IsPackable>false</IsPackable>
+    <ProjectName>Minsk</ProjectName>
```
Comment (centreboard) [inline src/msc/msc.csproj:7]: What does `<ProjectName>` control?

Comment (Nirmal4G) [reply src/msc/msc.csproj:7]: It's just a meta, but this is used here like this... Publish to a common directory.

https://github.com/terrajobst/minsk/blob/44be9e089142fb1b6ac113f4bb02b92563fbfd70/src/Directory.Build.targets#L19

**Comment by Nirmal4G** [comment]
I've marked as this a WIP, as there are some things needed to be fixed. Will update soon, hopefully before the stream.

**Comment by Nirmal4G** [comment]
- Removed other changes from PR, like package updates and build related.
- Made it easy for review as there are only formatting changes here.
- I'll rebase the PR on top of master after every episode to keep it up to date.

**Review by terrajobst — APPROVED** [review]

**Comment by Nirmal4G** [comment]
I have separated non-formatting changes into #131. I mention it here, just for reference.

**Pull request merged**